### ISSUE TITLE
Usage with classNames

### ISSUE THREAD

**Issue by ztanner** [atfzl/eslint-plugin-css-modules#18]
Thanks for creating this plugin! Super useful as I work on porting a project to CSS modules.

I'm running into an issue on components that make use of the [classNames library](https://github.com/JedWatson/classnames). The following syntax will cause the `css-modules/no-unused-class` error to trigger:

```jsx
const cx = classNames.bind(styles);
... 
<div className={cx('container', {
  'hidden': !this.state.isHidden
})} />
```

Is there a way around this? Thanks in advance.

**Comment by TrySound** [comment]
I guess. This case should handle classNames lib itself. All non-direct getters like `styles.container` can't be checked statically.

**Comment by atfzl** [comment]
Hi @ztanner,

Good to know you are finding this plugin useful.

It is not possible to statically analyse this pattern.
You have to change your code to this:
```js
import cx from 'classnames';
import s from './styles.css';
...
<div
  className={cx(s.container, {
    [s.hidden]: !this.state.isHidden,
  })}
/>
```

**Comment by ztanner** [comment]
Understood - I appreciate the alternative suggestion.
